src/include/sof/audio/src/src_config.h

```diff
@@ -29,13 +29,18 @@
 #if defined __XCC__
 #include <xtensa/config/core-isa.h>
 #define SRC_GENERIC	0
-#if XCHAL_HAVE_HIFI2EP == 1
-#define SRC_HIFIEP	1
+#if XCHAL_HAVE_HIFI4 == 1
+#define SRC_HIFI4	1
```

Comment:
if you set SRC_HIF4 to 0 outside, all you need is set the respective SRC_* in the corresponding  #if 's below

Reply:
We probably need a more global ```CONFIG_XTENSA_HIFI_VERSION``` that all processing could use and would mean this logic would only be needed in one place. This can be in another PR though.